Remotion/ObjectBinding/Web/UI/Controls/BocCustomColumnDefinition.cs

```diff
@@ -45,6 +45,7 @@ public class BocCustomColumnDefinition : BocColumnDefinition, IBusinessObjectCla
     private bool _isSortable;
     private bool _isRowHeader;
     private BocCustomColumnDefinitionMode _mode;
+    private bool _isVisible;
```

Comment:
I think we need to set this to `true`. I don't think that the `[DefaultValue(true)]` attributes does anything apart from designer stuff. As such, columns default to be invisible.